Big2/src/components/tabscreens/Rules.js

```diff
@@ -1,72 +1,82 @@
-</Text>
-{'/N'}
-<Text style={{fontWeight:"bold"}}>
-{'/N'}
-
-Single cards: Any card from the deck, ordered by rank with suit being the tie-breaker. (For instance, A♠ beats A♥, which beats K♠.)
-{'/N'}
-
-Pairs: Any two cards of matching rank, ordered as with singular cards by the card of the higher suit. (A pair consisting of the K♠ and K♣ beats a pair consisting of K♥ and K♦.)
-{'/N'}
-
-Triples: Three equal ranked cards, three twos are highest, then aces, kings, etc. down to three threes, which is the lowest triple. In some variations, a triple can only be played as part of a five-card hand.
-{'/N'}
-
-Five-card hands: There are five (var. 2) different valid five-card hands, ranking from low to high as follows (the same ranking as in poker, where applicable):
-{'/N'}
-
-Straight (also known as a snake in Cantonese or mokke in Malaysia): Any 5 cards in a sequence (but not all of the same suit). Rank is determined by the value of the biggest card, with the suit used only as a tie-breaker. Therefore "3-4-5-6-7" is less than "2-3-4-5-6", since 2 is considered the largest card in the 2-3-4-5-6 straight. The largest straight is A-2-3-4-5, second 2-3-4-5-6, third 10-J-Q-K-A while the smallest straight is 3-4-5-6-7.
-{'/N'}
-
-Flush (also known as a flower or sama bunga in Malaysia): Any 5 cards of the same suit (but not in a sequence). Rank is determined by Face value of the cards (highest first, then each lower card in order). Suit (♠,♥,♣,♦) is used to break ties. In some popular variations, flushes are not permitted as a playable hand, and thus it is the lowest possible combination.
-{'/N'}
 
-Full house: a composite of a three-of-a-kind combination and a pair. Rank is determined by the value of the triple, regardless of the value of the pair. Also known as a Fullen.
-{'/N'}
-
-Four-of-a-kind + One card (nicknamed King Kong, tiki, or Bomb or ampat batang in Malaysia): Any set of 4 cards of the same rank, plus any 5th card. (A four-of-a-kind cannot be played unless it is played as a five-card hand) Rank is determined by the value of the 4 card set, regardless of the value of the 5th card. It is also known as a poker. (Some play the Four-of-a-kind hand as the beat all, therefore nicknamed the bomb, King Kong, or also tiki.). In some variations, when a bomb was dealt, it immediately makes the 2s to be the lowest cards in the game (with 2 of diamonds becoming the lowest)
-{'/N'}
-
-Straight flush: A composite of the straight and the flush: five cards in sequence in the same suit. Ranked the same as straights, suit being a tie-breaker. (Sometimes also called a "bomb" (or tiki or sunn) in Malaysia, greater than a four-of-a-kind)
+<ScrollView style={styles.mainContainer}>
+
+<Text style={{backgroundImage:`url(${Background})`}}> 
```

Comment:
Some reason, on my machine, I keep getting an error when it tries to resolve this line of code:

`Warning: Failed prop type: Invalid props.style key `backgroundImage` supplied to `Text`.
Bad object: {
  "backgroundImage": "url(3)"
}`

I am going to have someone else check this to see if the parchment renders for them